### PR TITLE
Put proximity-dist_pin_to_airport table into dbt

### PR DESCRIPTION
**Pull request by mbjackson-capp**
Create SQL query for table that includes distance from O'Hare, distance from Midway, estimated DNL additions from each airport, and estimated total DNL (including baseline) for each airport. Update docs to include new model and new columns.

### CODE REVIEW

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
+-- CTAS to create a table of distance to the airports for each PIN
+
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:2]: **nitpick:**

```suggestion
```

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+{{
+  config(
+    materialized='table',
+    partitioned_by=['year'],
+    bucketed_by=['pin10'],
+    bucket_count=1
+  )
+}}
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:10]: **nitpick:** For consistency with the other CTAS:

```suggestion
{{
    config(
        materialized='table',
        partitioned_by=['year'],
        bucketed_by=['pin10'],
        bucket_count=1
    )
}}
```

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            ST_POINT(dp.x_3435, dp.y_3435),
+            ST_POINT(1142843, 1864827) --midway centroid
+        ) AS dist_to_midway        
+    FROM distinct_pins AS dp
+),
+
+--COEFFICIENTS FROM OMP-ONLY inverse square noise falloff model for O'Hare,
+-- 2019-sensor data ONLY inverse square noise falloff model for Midway:
+--ohare     0.00582564978995262
+--midway    0.00297149980129393
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:37]: **suggestion:** Let's link to the R script on GitHub that generates these coefficients (once #193 is merged).

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+airport_modeled_dnl AS (
+    SELECT 
+        *,
+        GREATEST(
+            0,
+            10 * LOG(10, (model_output_ohare / POWER(10,-12)))
+        ) AS dnl_ohare,
+        GREATEST(
+            0,
+            10 * LOG(10, (model_output_midway / POWER(10,-12)))
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:61]: **nitpick:** Formatting:

```suggestion
            10 * LOG(10, (model_output_ohare / POWER(10, -12)))
        ) AS dnl_ohare,
        GREATEST(
            0,
            10 * LOG(10, (model_output_midway / POWER(10, -12)))
```

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        ) AS dnl_midway
+    FROM airport_regression
+)
+
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:65]:
```suggestion
```

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    dnl.dnl_ohare,
+    dnl.dnl_midway,
+    dnl.dnl_ohare + dnl.dnl_midway + 50 AS sum_airport_dnl
+FROM {{ source('spatial', 'parcel') }} AS pcl
+INNER JOIN airport_modeled_dnl AS dnl
+    ON pcl.x_3435 = dnl.x_3435
+    AND pcl.y_3435 = dnl.y_3435
+ORDER BY dnl.dnl_ohare DESC 
+LIMIT 1000
+
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:81]: **issue:** You'll need to remove this for the final PR.

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+            0,
+            10 * LOG(10, (model_output_midway / POWER(10,-12)))
+        ) AS dnl_midway
+    FROM airport_regression
+)
+
+
+SELECT
+    pcl.pin10,
+    pcl.year, --do we want this?
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:69]: **suggestion:** Move questions comments like this one to PR comments.

But yes, we do want this included in the output. Note that it's partition field in the `config` block at the top of this query. That means year is used to "chunk" the data such that a separate parquet file is saved for each year on S3. It also required year to be the last column for some reason, see the error in the workflow run that tried to generate this CTAS: https://github.com/ccao-data/data-architecture/actions/runs/6868698948/job/18679860499?pr=231

So basically, just move this column below L74.

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+
+SELECT
+    pcl.pin10,
+    pcl.year, --do we want this? 
+    dnl.dist_to_ohare,
+    dnl.dist_to_midway,
+    dnl.dnl_ohare,
+    dnl.dnl_midway,
+    dnl.dnl_ohare + dnl.dnl_midway + 50 AS sum_airport_dnl
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:74]: For the sake of consistency with other `proximity` CTAS, let's rename these columns and add a "data year":

```suggestion
    dnl.airport_ohare_dist_ft,
    dnl.airport_midway_dist_ft,
    dnl.airport_dnl_ohare,
    dnl.airport_dnl_midway,
    dnl.dnl_ohare + dnl.dnl_midway + 50 AS airport_dnl_total,
    '2019' AS airport_data_year
```

File: dbt/models/proximity/columns.md [hunk 9]
```diff
@@ -7,6 +7,40 @@ Schools of any type (elementary, secondary, etc.) are included.
 School ratings sourced from [GreatSchools](https://www.greatschools.org/)
 {% enddocs %}
 
+## dist_to_midway
+
+{% docs column_dist_to_midway %}
+Distance to centroid of Midway airport (feet).
+
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:14]: **nitpick:** For consistent formatting with other CTAS:

```suggestion
Distance to centroid of Midway airport (feet)
```

File: dbt/models/proximity/columns.md [hunk 10]
```diff
@@ -7,6 +7,40 @@ Schools of any type (elementary, secondary, etc.) are included.
...
+{% enddocs %}
+
+## dist_to_ohare
+
+{% docs column_dist_to_ohare %}
+Distance to centroid of O'Hare airport (feet).
+
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:21]: **nitpick:** For consistent formatting with other CTAS:

```suggestion
Distance to centroid of O'Hare airport (feet)
```

File: dbt/models/proximity/columns.md [hunk 11]
```diff
@@ -7,6 +7,40 @@ Schools of any type (elementary, secondary, etc.) are included.
 School ratings sourced from [GreatSchools](https://www.greatschools.org/)
 {% enddocs %}
 
+## dist_to_midway
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:10]: Be sure to rename the column names here to match the changes in the CTAS.

File: dbt/models/proximity/columns.md [hunk 12]
```diff
@@ -7,6 +7,40 @@ Schools of any type (elementary, secondary, etc.) are included.
...
+
+{% docs column_dist_to_midway %}
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:12]: And rename them here.

File: dbt/models/proximity/columns.md [hunk 13]
```diff
@@ -7,6 +7,40 @@ Schools of any type (elementary, secondary, etc.) are included.
...
+{% enddocs %}
+
+## dnl_midway
+
+{% docs column_dnl_midway %}
+Estimated contribution of Midway airport to PIN. Measured in decibels; to be
+interpreted as contribution to DNL (Day-Night Level) estimate of 24-hour average
+of decibel level. Produced by modeling noise level as inverse-square falloff 
+from a point source at the centroid of the airport.
+
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:31]: **nitpick:** For consistent formatting with other CTAS:

```suggestion
Estimated contribution of Midway airport to PIN noise.

Measured in decibels; to be interpreted as contribution to DNL
(Day-Night Level) estimate of 24-hour average of decibel level.
Produced by modeling noise level as inverse-square falloff 
from a point source at the centroid of the airport.
```

File: dbt/models/proximity/columns.md [hunk 14]
```diff
@@ -7,6 +7,40 @@ Schools of any type (elementary, secondary, etc.) are included.
...
+{% enddocs %}
+
+## dnl_ohare
+
+{% docs column_dnl_ohare %}
+Estimated contribution of O'Hare airport to PIN. Measured in decibels; to be
+interpreted as contribution to DNL (Day-Night Level) estimate of 24-hour average
+of decibel level. Produced by modeling noise level as inverse-square falloff 
+from a point source at the centroid of the airport.
+
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:41]: **nitpick:** For consistent formatting with other CTAS:

```suggestion
Estimated contribution of O'Hare airport to PIN noise.
Measured in decibels; to be interpreted as contribution to DNL
(Day-Night Level) estimate of 24-hour average of decibel level.
Produced by modeling noise level as inverse-square falloff 
from a point source at the centroid of the airport.
```

File: dbt/models/proximity/columns.md [hunk 15]
```diff
@@ -380,3 +414,14 @@ Number of schools (any kind) within half mile.
...
+## sum_airport_dnl
+
+{% docs column_sum_airport_dnl %}
+Estimated DNL for a PIN, assuming a baseline DNL of 50 ("quiet suburban") and
+adding predicted noise from O'Hare and Midway airports to that baseline.
+
+For more on DNL, see 
+https://www.faa.gov/regulations_policies/policy_guidance/noise/basics
+
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:426]:
```suggestion
```

**Review by dfsnow — COMMENTED** [review]
This looks great @mbjackson-capp. Very close to done. Just make the edits in this PR and correct the linting errors in [this workflow](https://github.com/ccao-data/data-architecture/actions/runs/6868698951/job/18679860184?pr=231) and it should be all set.

File: dbt/models/proximity/columns.md [hunk 16]
```diff
@@ -1,3 +1,54 @@
...
+Measured in decibels; to be interpreted as contribution to DNL
+(Day-Night Level) estimate of 24-hour average of decibel level.
+Produced by modeling noise level as inverse-square falloff 
+from a point source at the centroid of the airport.
+{% enddocs %}
+
+## airport_dnl_ohare
+
+{% docs column_airport_dnl_ohare %}
+Estimated contribution of O'Hare airport to PIN noise.
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:23]:
```suggestion
Estimated contribution of O'Hare airport to PIN noise.

```

File: dbt/models/proximity/columns.md [hunk 17]
```diff
@@ -1,3 +1,54 @@
+## airport_data_year
+
+{% docs column_airport_data_year %}
+Set to year 2019 for all PINs. Note that O'Hare noise predictions 
+are built from O'Hare Modernization Program (OMP) projected values.
+
```
Comment (dfsnow) [inline dbt/models/proximity/columns.md:6]:
```suggestion
```

File: aws-athena/ctas/proximity-dist_pin_to_airport.sql [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+        GREATEST(
+            0,
+            10 * LOG(10, (model_output_ohare / POWER(10, -12)))
+        ) AS airport_dnl_ohare,
+        GREATEST(
+            0,
+            10 * LOG(10, (model_output_midway / POWER(10, -12)))
+        ) AS airport_dnl_midway
+    FROM airport_regression
+)
```
Comment (dfsnow) [inline aws-athena/ctas/proximity-dist_pin_to_airport.sql:67]:
```suggestion
)

```

**Review by dfsnow — APPROVED** [review]
